README.md

```diff
@@ -41,6 +143,5 @@ Hello world program
 
 This project is [MIT](./LICENSE) licensed.
 
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
+```
```

Comment:
- [ ] I notice that your MIT license is broken please fix this license so the README file looks professional. Here is the [LInk](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository) how you can create license in your README file.

Reply:
@roshan-bajgain 
I've fixed the broken link and used the MIT license.
Thanks